dockstore-integration-testing/src/test/java/io/dockstore/client/cli/RefreshByOrgIT.java

```diff
@@ -0,0 +1,203 @@
+package io.dockstore.client.cli;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.TimeoutException;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.Response;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dockstore.common.CommonTestUtilities;
+import io.dockstore.webservice.DockstoreWebserviceApplication;
+import io.dockstore.webservice.DockstoreWebserviceConfiguration;
+import io.dockstore.webservice.core.Tool;
+import io.dockstore.webservice.core.Workflow;
+import io.dropwizard.client.JerseyClientBuilder;
+import io.dropwizard.setup.Environment;
+import io.dropwizard.testing.ResourceHelpers;
+import io.dropwizard.testing.junit.DropwizardAppRule;
+import io.swagger.client.ApiException;
+import org.apache.commons.dbutils.handlers.ScalarHandler;
+import org.glassfish.jersey.client.ClientProperties;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import static io.dockstore.common.CommonTestUtilities.clearStateMakePrivate2;
+import static io.dockstore.common.CommonTestUtilities.getTestingPostgres;
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * @author gluu
+ * @since 25/09/17
+ */
+public class RefreshByOrgIT {
+
+    @ClassRule
+    public static final DropwizardAppRule<DockstoreWebserviceConfiguration> RULE = new DropwizardAppRule<>(
+            DockstoreWebserviceApplication.class, ResourceHelpers.resourceFilePath("dockstoreTest.yml"));
+    private static Client client;
+    private static String token;
+    private static String usersURLPrefix;
+    private static ObjectMapper objectMapper;
+    private static Long id;
+
+    @BeforeClass
+    public static void clearDBandSetup() throws IOException, TimeoutException, ApiException {
+        clearStateMakePrivate2();
+        final CommonTestUtilities.TestingPostgres testingPostgres = getTestingPostgres();
+        id = testingPostgres.runSelectStatement("select id from enduser where username='DockstoreTestUser2';", new ScalarHandler<>());
+        Environment environment = RULE.getEnvironment();
+        token = testingPostgres.runSelectStatement("select content from token where tokensource='dockstore';", new ScalarHandler<>());
+        client = new JerseyClientBuilder(environment).build("test client").property(ClientProperties.READ_TIMEOUT, 60000);
```

Comment:
comment on why the 60000 value changes a couple times in this sequence of commits